release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -464,6 +464,14 @@ For more information, see xref:../updating/preparing_for_updates/updating-cluste
 [id="ocp-release-notes-machine-management_{context}"]
 === Machine management
 
+[id="ocp-4-19-cpms-prefix_{context}"]
+==== Custom prefixes for machine names in the control plane machine set
```

Comment:
This line seems a bit strange to me. How about changing to `Custom prefixes for control plane machine names`? But I'm not sure if this is better, and I'd like to hear other people's opinions.

Reply:
Sure thing. It seems like @chiragkyal gave your suggestion a +1, so I will make that change!